rainbows.rb

```diff
@@ -25,5 +26,21 @@ def mix_colors(color1,color2)
   return mixed_color
 end
 
+rainbow = [{ red: { r:255,g:0,b:0 }},
+          { orange: { r:255, g:127, b:0 }},
+          { yellow: { r:255, g:255, b:0 }},
+          { green: { r:0, g: 255, b:0 }},
+          { blue: { r:0, g: 0, b:255 }},
+          { violet: { r: 125, g:0, b:255 }},
+          { magenta: { r:255, g:0, b:255 }}]
```

Comment:
nit-picky formatting: put []'s for array on separate lines from hashes, which allows easier deleting/commenting out of individual elements in the array.